keyboards/giabalanaipico/readme.md

```diff
@@ -0,0 +1,24 @@
+# giabalanaipico
+
+<p align=center>
+<img width="700" alt="giabalanaipico_logo" src="https://github.com/3araht/giabalanaipico/blob/master/pictures/giabalanaipico_logo.png">
+</p>
+
+![giabalanaipico overview](https://github.com/3araht/giabalanaipico/blob/master/pictures/giabalanaipico_overview.jpg)
```

Comment:
Please don't use HTML tags.  Use markdown.

Also, please rehost the images on Imgur.